object-manipulation-2/src/omit.js

```diff
@@ -0,0 +1,19 @@
+/* exported omit */
+function omit(source, keys) {
+  var omitObject = {};
+
+  if (Object.keys(source).length !== 0) {
+    for (var i in source) {
+      omitObject[i] = source[i];
+    }
+
+    for (var keysIndex = 0; keysIndex < keys.length; keysIndex++) {
+      for (var j in omitObject) {
+        if (keys[keysIndex] === j) {
+          delete omitObject[j];
+        }
+      }
+    }
```

Comment:
Try this approach to simplify things a bit:

- for each `sourceKey` in `source`
  - for each `keys` at `i`
    - declare a boolean for whether this `sourceKey` is `toBeOmitted`
    - if `sourceKey` is equal to `keys` at `i`, then
      - this `sourceKey` is `toBeOmitted`
      - `break` out of this inner loop
  - if `sourceKey` is not `toBeOmitted`, then
    - assign `source` at `sourceKey` to your new object at `sourceKey`.